### ISSUE TITLE
Labels

### ISSUE THREAD

**Issue by adagj** [NorESMhub/noresm_simulations#5]
I suggest different (or additional) labels: Short compset names. We'll have N1850 for fully coupled piControl, NF1850 for amip style piControl, NHIST for fully coupled historical, NFHIST for amip style historical ect. I actually don't think NorESM is a good label. Feel free to think differently :)

**Comment by TomasTorsvik** [comment]
I would opt for additional labels, an issue can have multiple. Agree that `NorESM` is not very useful in this setting.

**Comment by oyvindseland** [comment]
I agree but it should not be overdone. There are very many compsets so they often have to be grouped together.

**Comment by mvertens** [comment]
@adagj @TomasTorsvik - what is the conclusion with this issue? Do we still want to have it open?

**Comment by TomasTorsvik** [comment]
I suggest to close this issue, feel free to re-open if thre is more to discuss.

The labels we have at the moment seems sufficient for the development work we are doing. I'm not sure it would be helpful to try to enforce some more rigid standard.
